packages/auth/__tests__/selling-partner-api-auth.spec.ts

```diff
@@ -90,7 +90,7 @@ describe('src/selling-partner-api-auth', () => {
     })
   })
 
-  it('should failed for missing secretAccessKey', () => {
+  it('should not failed for missing secretAccessKey', () => {
```

Comment:
```suggestion
  it('should not fail for missing secretAccessKey', () => {
```

While you’re at it :p